common-test/src/main/java/io/strimzi/test/StrimziRunner.java

```diff
@@ -470,4 +474,35 @@ private KubeClusterResource clusterResource() {
         }
         return clusterResource;
     }
+
+    private Statement withTopic(Annotatable element, Statement statement) {
+        Statement last = statement;
+        for (Topic cluster : annotations(element, Topic.class)) {
+            String yaml = getContent(new File(TOPIC_CM), node -> {
+                JsonNode metadata = node.get("metadata");
+                ((ObjectNode) metadata).put("name", String.valueOf(cluster.name()));
+                JsonNode labels = metadata.get("labels");
+                ((ObjectNode) labels).put("strimzi.io/cluster", String.valueOf(cluster.clusterName()));
+                JsonNode data = node.get("data");
+                ((ObjectNode) data).put("name", String.valueOf(cluster.name()));
+                ((ObjectNode) data).put("partitions", String.valueOf(cluster.partitions()));
+                ((ObjectNode) data).put("replicas", String.valueOf(cluster.replicas()));
+            });
+            last = new Bracket(last) {
+                @Override
+                protected void before() {
+                    LOGGER.info("Creating Topic", cluster.name(), name(element));
+                    // create cm
+                    kubeClient().createContent(yaml);
+                }
+                @Override
+                protected void after() {
+                    LOGGER.info("Deleting ConfigMap '{}' after test per @Topic annotation on {}", cluster.clusterName(), name(element));
+                    // delete cm
+                    kubeClient().deleteContent(yaml);
```

Comment:
The same story, should we check that CM was deleted? 
an example : `kubeClient().waitForResourceDeletion("CM", cnName); `